### DISCUSSION TITLE
Signac and ArchR

### DISCUSSION THREAD

**Discussion by drosop** [GreenleafLab/ArchR#2002] [Questions / Documentation]
Hi,

I have processed my 10X multiome data using signac, and I have signac object containing cluster annotations, count matrix, etc.

I want to use some features of archR for eg, creating pseudobulk replicates ( addGroupCoverage()) and other downstream analysis. 

Is there any way to use signac object in archR? 

 1. Convert signac object to archR project  (I found archRtoSignac but I need to convert SignactoarchR) 
 2. or Any other feasible way?

I am newbie to this analysis. Any help will be greatly appreciated.

Thank you so much,

**Comment by rcorces** [comment]
We dont provide code to do this. Sorry

**Reply by drosop** [reply]
Thank you for your prompt reply.

Is there any existing way to achieve this?
